content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -14,7 +14,19 @@ You've deployed your application and exposed it via a service. Now what? Kuberne
 
 ## Organizing resource configurations
 
-Many applications require multiple resources to be created, such as a Deployment and a Service. Management of multiple resources can be simplified by grouping them together in the same file (separated by `---` in YAML). For example:
+Many applications require multiple resources to be created, such as a Deployment and a Service. Management of multiple resources can be simplified by grouping them together in the same file.
+Techniques for grouping together resources in YAML include separating:
+
+ - Different resource types with hyphens (`---`)
```

Comment:
You always need to separate objects in a single YAML file with _document markers_ such as `---`, so let's emphasize that.

Reply:
+1 please call it a document separator.

Reply:
Okay. :-)

Reply:
I removed my changes to this file.